packages/grid/x-grid-modules/src/components/styled-wrappers/styles.module.css

```
@@ -0,0 +1,223 @@
+.gridRoot {
+    box-sizing: border-box;
+}
+.gridRoot * {
+    box-sizing: border-box;
+}
+
+.gridRoot {
+    position: relative;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu,
+    Cantarell, 'Helvetica Neue', sans-serif;
+    border: 1px solid #bdc3c7;
+    border-radius: 4px;
+    outline: none;
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+}
+.gridRoot .main-grid-container {
+    position: relative;
+    flex-grow: 1;
+    flex-direction: column;
+    display: flex;
+}
+.gridRoot .watermark {
+    position: absolute;
+    pointer-events: none;
+    color: #8282829e;
+    z-index: 100000;
+    width: 100%;
+    text-align: center;
+    bottom: 50%;
+    right: 0;
+    letter-spacing: 5px;
+    font-size: 24px;
+}
+.gridRoot .footer {
```

I would **highly** recommend we flatten all the CSS selectors to an as low as possible specificity. For instance, in this case:

```suggestion
.MuiDataGrid-footer {
```